packages/typespec-test/test/ai/generated/typespec-ts/src/index.ts

```diff
@@ -245,6 +244,7 @@ export {
   RunAdditionalFieldList,
   VectorStoreFileStatusFilter,
 } from "./models/agents/index.js";
+export { AIProjectClient } from "./aiProjectClient.js";
```

Comment:
This happens a lot, @qiaozha can we priotize this task: https://github.com/Azure/autorest.typescript/issues/2923?